mantidimaging/gui/windows/image_load_dialog/test/presenter_test.py

```diff
@@ -23,13 +23,23 @@ def setUp(self):
         self.p = LoadPresenter(self.v)
 
     def test_do_update_sample_with_no_selected_file(self):
-        self.v.select_file.return_value = None
+        self.p.do_update_sample(None)
 
-        self.p.do_update_sample()
-
-        self.v.select_file.assert_called_once_with("Sample")
         self.v.ok_button.setEnabled.assert_called_once_with(False)
 
+    @mock.patch("mantidimaging.gui.windows.image_load_dialog.presenter.LoadPresenter.do_update_flat_or_dark")
+    def test_do_update_field(self, mock_do_update_flat_or_dark):
+        selected_file = "SelectedFile"
+        self.v.select_file.return_value = selected_file
+        name = "Name"
+        suffix = "Apples"
```

Comment:
NITPICK: As there is only one suffix, should we rename this from Apples 🍎 🍏 to Apple 🍎 ? 😉

Reply:
These go away in the next PR.

Reply:
I'll miss the Apple suffix 😢